src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/CodeGenerator.cs

```diff
@@ -148,15 +143,11 @@ private void InitILGeneration(string methodName, Type[] argTypes)
             _argList = new List<ArgBuilder>();
             for (int i = 0; i < argTypes.Length; i++)
                 _argList.Add(new ArgBuilder(i, argTypes[i]));
-            if (_codeGenTrace != CodeGenTrace.None)
-                EmitSourceLabel("Begin method " + methodName + " {");
         }
```

Comment:
Is there a reason to remove the tracing?

Reply:
It was never enabled. _codeGenTrace is set to None in the constructor.